zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestCamundaApplication.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.qa.util.cluster;
+package io.camunda.zeebe.qa.util.cluster;
```

Comment:
❌ ❓ Sorry maybe I missed something, but why you're moving the classes back to the Zeebe qa modules?

Reply:
It was not clear which folders it should be, because the qa module on the root was not functional at all, so the reason I moved the to the folder that was funcitonal, I guess was a missunderstanding for my part

Because half of tests (reading by the CI) was on Zeebe/zeebe, and only the Standalone Camunda App was on the root, but not correctly read by intellij

In order to clarify, which folder should has this? So I can move it and keep all on just one, or should we have all the tests over the root?  (It is not really clear on my side)

Reply:
It would be good for the next time if we could clarify this upfront :) so please if in doubt ask.

The qa on root level was added via https://github.com/camunda/camunda/pull/19757 and mentioned in several updates in slack https://camunda.slack.com/archives/C06HTSPD5AP/p1719472381357659


>  because the qa module on the root was not functional at all

Well that is not true. Tests are executed on CI, you can see for example on recent main change https://github.com/camunda/camunda/actions/runs/9776526029/job/26989329015

[An successful run](https://github.com/camunda/camunda/actions/runs/9776526029/job/26989329015#step:9:118)

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.camunda.it.StandaloneCamundaTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 55.24 s -- in io.camunda.it.StandaloneCamundaTest
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
```